docs/guidelines/postman.md

```diff
@@ -0,0 +1,122 @@
+
+Assuming 
+* one has exported the desired collection to a `collection.json` file, 
+* set up [Postman API Key 🔗](https://learning.postman.com/docs/developer/postman-api/authentication/#generate-a-postman-api-key)
+* configured `POSTMANAPIKEY` as a github action secret
+* [obtained 🔗](https://support.postman.com/hc/en-us/articles/5063785095319-How-to-find-the-ID-of-an-element-in-Postman) the `collectionId`
+
+a GitHub Action workflow _could_ look something like:
+``` YAML
+name: Running API tests
+run-name: ${{ github.actor }} is testing the API
+on: [pull_request]
+jobs:
+    api_tests:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Set up Node
+              uses: actions/setup-node@v4
+            - name: Install Newman
+              run: npm install -g newman 
+            - name: Run API tests
+              run: newman run "https://api.getpostman.com/collections/${{collectionID}}?apikey=${{ secrets.POSTMANAPIKEY }}" --environment "https://api.getpostman.com/environments/${{ environmentID }}?apikey=${{ secrets.POSTMANAPIKEY }}"
+            
+```
+
+Some considerations on this workflow:
+
+* [The Postman API is rate-limited](https://learning.postman.com/docs/developer postman-api/postman-api-rate-limits/), so keep that in mind when running automatic workflows.
+* The generated `POSTMANAPIKEY` should be considered as a PAT - it gives the workflow the same access as _your_ account.
+* This example uses the enivornment obtained through the Postman API _as-is_. Consider if the environment could contain sensitive data (e.g. client secrets). These entries could then instead be placed in a secure Vault (e.g. Azure KeyVault), which would introduce some extra steps in the workflow; extracting Postman environment in separate step, logging into Azure, extracting data from the KeyVault, [using secrets as input parameters 🔗](https://github.com/postmanlabs/newman?tab=readme-ov-file#newman-run-collection-file-source-options) to Newman
+
+
+## Lightweight API Client
+The "old" lightweight API client, based on "Scratch Pad", has been discountinued and there is only a single official API Client that can be downloaded.
+After downloading and executing the file, there is a choice of `Or continue with the lightweight API client.` (as of April 2024, this represents itself as a small text below the login-button on starting the application). Running the application in this mode will ensure that everything stays local - nothing is being synchronized to the cloud. However, it will not be possible to use the full set of features Postman provides and the usage in this mode will be limited to simple manual API testing.
+
+!!! info
+    Using the Lightweight client is free and one does not need to apply for access in AccessIT. In many cases this will be sufficient.
```

Comment:
Perhaps we should make this a bit more specific stating that it's not necessary to apply for a Enterprise account

Reply:
👍